### PR TITLE
Fix foreman bootstrap script

### PR DESCRIPTION
**Pull request by jnamster**
Encountered error that it could not find webpack when starting the server. 
Change made in Procfile.dev so it installed dependencies before the build.

### CODE REVIEW

**Comment by HashNotAdam** [comment]
This branch relies on the Secret Share branches so we need to approve those first

**Comment by HashNotAdam** [comment]
@julianguyen there is a huge amount of complexity added by that React branch, have you thought any more about simplifying the system? Obviously, I suggested guard but the specific answer doesn't matter so much as making life easy for people.

**Comment by baohouse** [comment]
Perhaps you can create a clean branch with just the `yarn install` amendment?

File: db/schema.rb [hunk 1]
```diff
@@ -141,6 +141,9 @@
     t.boolean  "comment"
     t.text     "strategy"
     t.string   "slug"
+    t.uuid     "secret_share_identifier"
```
Comment (julianguyen) [inline db/schema.rb]: Looks like this file was accidentally committed?

**Pull request merged**